admin_console/forms/PartnerConfiguration.php

```diff
@@ -359,6 +359,17 @@ public function init()
 			'filters'		=> array('StringTrim'),
 		));
 		
+		$this->addElement('button', 'password_structure_validations_edit', array(
+			'label'		=> 'Edit Rules',
+			'decorators'	=> array('ViewHelper'),
+		));
+		$this->getElement('password_structure_validations_edit')->setAttrib('onClick', 'openPasswordValidationRules()');
+		
+		$this->addElement('hidden', 'password_structure_validations', array(
+			'label'		=> 'password_structure_validations',
```

Comment:
why the label is not written as simple text?

Reply:
Label unused, can be removed